### DISCUSSION TITLE
Instructions for updateing from 0.9.0 to 0.9.4 on Home Assistant

### DISCUSSION THREAD

**Discussion by kdfha** [blakeblackshear/frigate#2216] [Ask A Question]
I was having a problem with recordings missing the event and Blake asked why I wasn't on 0.9.4. 

I installed it as an add-on on HA on a RPi 4 using the repository and that's what I got with no options to update.

I don't know how to update it. I have searched for updating Frigate NVR and I don't find any instructions to do so. 

Can someone instruct me?

Should it be in the Documentation?

Thanks,
Ken

**Comment by blakeblackshear** [comment]
You could try removing and re-adding the repository. I'm not sure why you aren't seeing the option to update.

**Reply by kdfha** [reply]
Removing and re-adding the repository got it up to 0.9.4 so that is good. 

There still isn't anything about updating though. 

The only options are:

Start on boot
Watchdog
Show in sidebar

and in the Full Access version:

Protection mode

I'll try to keep an eye on when you update it on the site. 

Thanks
